content/soarian/dstu2/medications/immunization.md

```diff
@@ -31,15 +31,18 @@ _Implementation Notes_
 
 * The API returns the most current immunization list.
 
+* The API returns a consolidated list of immunizations charted as complete.
```

Comment:
Updated the documentation as 'The API returns a consolidated list of those immunizations that are charted as complete'.
Regarding 'Complete' status we are going to fix the code before certification.